apk-generator/v2/app/generator/generator.py

```diff
@@ -221,6 +221,7 @@ def download_event_data(self):
         self.save_file_in_temp_assets('speakers')
         self.save_file_in_temp_assets('sponsors')
         self.save_file_in_temp_assets('tracks')
+        self.save_file_in_temp_assets('sessions/types')
         logger.info('Download complete')
```

Comment:
@iamareebjamal there is no indentation error. i know it is showing wrong here but i dont know the reason for it.
i think its fine now.